apps/docs/pages/guides/auth/sso/auth-sso-saml.mdx

```diff
@@ -0,0 +1,314 @@
+- **RelayState**  
+  State used by Supabase Auth to hold information about a request to verify the identity of a user.
+
+## Important SAML 2.0 information
+
+Below is information about your project's SAML 2.0 configuration which you can share with the company or organization that you're trying to on-board.
+
+| Name | Value |
+| ---- | ----- |
+| EntityID | `https://<project>.supabase.co/auth/v1/sso/saml/metadata` |
+| Metadata URL | `https://<project>.supabase.co/auth/v1/sso/saml/metadata` |
+| ACS URL | `https://<project>.supabase.co/auth/v1/sso/saml/acs` |
+| SLO URL | `https://<project>.supabase.co/auth/v1/sso/slo` |
+| NameID | Required `emailAddress` or `persistent` |
+
+Note that SLO (Single Logout) is not supported at this time with Supabase Auth as it is a rarely supported feature by identity providers. However the URL is registered and advertised for when this does become available.
+
+Append `?download=true` to the Metadata URL to get a downloadable Metadata XML file.
+
+### User accounts and identities
+
+User accounts and identities created via SSO differ from regular (email, phone, password, social login...) accounts in these ways:
+
+- **No automatic linking.**  
+  Each user account verified using a SSO identity provider will not be automatically linked to existing user accounts in the system. That is, if a user `jane.doe@company.com` had signed up with a password, and then uses their company SSO login with your project, there will be two `jane.doe@company.com` user accounts in the system.
+- **Emails are not necessarily unique.**  
+  Given the behavior with no automatic linking, email addresses are no longer a unique identifier for a user account. Please always use the user's UUID to correctly reference user accounts.
+- **Sessions may have a maximum duration.**  
+  Depending on the configuration of the identity provider, a login session established with SSO may forcibly log out a user after a certain period of time.
+
+### Row Level Security
+
+You can use information about the SSO identity provider in Row Level Security policies.
+
+Here are some commonly used statements to extract SSO related information from the user's JWT:
+
+- `auth.jwt()#>>'{amr,0,method}'`
+  Returns the name of the last method used to verify the identity of this user. With SAML SSO this is `sso/saml`.
+- `auth.jwt()#>>'{amr,0,provider}'`
+  Returns the UUID of the SSO identity provider used by the user to sign-in.
+- `auth.jwt()#>>'{user_metadata,iss}'`
+  Returns the identity provider's SAML 2.0 `EntityID`
+
+<Admonition type="caution">
+If you use [Multi-Factor Authentication](/guides/auth/auth-mfa) with SSO, the `amr` array may have a different method at index `0`!
+</Admonition>
```

Comment:
Right now you can't actually use MFA with SSO, GoTrue will reject the request, but this is only temporary and will be possible with general availability.